MAUI/Funnel-Charts/DataLabels.md

```diff
@@ -91,4 +91,73 @@ chart.DataLabelSettings.LabelStyle = new ChartDataLabelStyle()
 
 {% endtabs %} 
 
-![Data label for .NET MAUI Funnel chart](DataLabel_images/MAUI_data_label.png)
\ No newline at end of file
+![Data label for .NET MAUI Funnel chart](DataLabel_images/MAUI_data_label.png)
+
+## LabelTemplate
+
+The [SfFunnelChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfFunnelChart.html) provides support to customize the appearance of the datalabel by using the [LabelTemplate]() property.
```

Comment:
The [SfFunnelChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfFunnelChart.html) provides support to customize the appearance of the datalabel using the [LabelTemplate]() property.

Reply:
Content updated.